Tasks/AzureFileCopyV3/task.loc.json

```diff
@@ -14,7 +14,7 @@
   "version": {
     "Major": 3,
     "Minor": 229,
-    "Patch": 1
+    "Patch": 2
```

Comment:
Seems code generator doesn't allow to change minor versions for generated tasks by spring tag